File: src/me/ch2/cmptext.py
from pathlib import Path

root = Path(__file__).parent
afile = root / "fizzbuzz_python.txt"
bfile = root / "fizzbuzz_rust.txt"

with open(afile, "r") as f:
    a = f.read().strip()
with open(bfile, "r") as f:
    b = f.read().strip()

if a == b:
    print("OK")
else:
    print('ng')

File: src/me/ch2/cmptext.rs
fn main() {
    let afile = "./fizzbuzz_python.txt";
    let bfile = "./fizzbuzz_rust.txt";

    // file as string
    let afile_str = std::fs::read_to_string(afile).expect("Unable to read file");
    let bfile_str = std::fs::read_to_string(bfile).expect("Unable to read file");

    // trim
    let afile_str = afile_str.trim();
    let bfile_str = bfile_str.trim();

    if afile_str == bfile_str {
        println!("Files are the same");
    } else {
        println!("Files are different");
    }
}
